runtime/cudaq/photonics.h

```diff
@@ -0,0 +1,11 @@
+/****************************************************************-*- C++ -*-****
+ * Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+#pragma once
+
+#include "qis/managers/photonics/photonics_qis.h"
```

Comment:
The more I think on this, the more I think we should specialize the naming used here. Instead of `photonics` I think it should be more specific to the target, like `orca` or `orca-tbi` or `orca-tbi-sim`. I think that photonics is too general and CUDA Quantum will likely interface with other photonics-based quantum computing setups in the future.